src/hooks/folderPageHooks.js

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { FOLDER_PAGE_API_URL } from "../constants/constants";
+
+export function useFolderPageUser(){
+  const [user, setUser] = useState();
+
+  const getUser = async() => {
+    try{
+      const response = await axios.get(`${FOLDER_PAGE_API_URL}/1`);
+      setUser(response.data.data[0]);
+    } catch(error) {
+      alert(error)
+    }
+  }
+
+  useEffect(() => {
+    getUser();
+  }, [])
+
+  return user;
+}
+
+export function useFolderPageFolder(){
+  const [folder, setFolder] = useState([]);
```

Comment:
내용을 보니 특정 유저의 폴더목록을 조회하는 것이군요.
이렇게 제안드립니다.
```suggestion
export function useUserFolders( userId ){ // or useUserFolderList
  const [folders, setFolders] = useState([]);
```